src/test/java/com/thealgorithms/maths/FactorialTest.java

```diff
@@ -5,9 +5,19 @@
 import org.junit.jupiter.api.Test;
 
 public class FactorialTest {
+    private static final String EXCEPTION_MESSAGE = "Input number cannot be negative!";
```

Comment:
```suggestion
    private static final String EXCEPTION_MESSAGE = "Input number cannot be negative";
```
I wouldn't be so emotional in the exception message :)

Reply:
Ok, no emotions next time.